src/Manifolds.jl

```diff
@@ -247,7 +247,6 @@ using ManifoldsBase:
     @invoke_maker,
     _euclidean_basis_vector,
     combine_allocation_promotion_functions,
-    default_inverse_retraction_method,
```

Comment:
where did this line go? I would prefer not to delete that.

Reply:
`default_inverse_retraction_method` is actually imported in line 27. This `using` confused me at one point because I needed to add a method to this function but this `using` overrides the `import` statement and disallows adding new methods without specifying module.

Reply:
oh, I dod not see that. We should be more careful here in the future then that import and using are disjoint (and ordered alphabetically).